code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm

```diff
@@ -38,8 +38,7 @@
 	anchors += locate(x+2,y-2,z)
 
 	for(var/turf/T in anchors)
-		var/datum/beam/B = Beam(T, "vine", time=INFINITY, maxdistance=5, beam_type=/obj/effect/ebeam/vine)
-		B.sleep_time = 10 //these shouldn't move, so let's slow down updates to 1 second (any slower and the deletion of the vines would be too slow)
+		Beam(T, "vine", maxdistance=5, beam_type=/obj/effect/ebeam/vine)
```

Comment:
Can we do anything to achieve this lack of movement?

Reply:
?

Reply:
happen again, see the comment ` //these shouldn't move, so let's slow down updates to 1 second (any slower and the deletion of the vines would be too slow`